src/input/AxisConfigWidget.cc

```diff
@@ -51,11 +51,24 @@ void AxisConfigWidget::AxesChanged(int nr, double val) const{
 	if(progressBar==nullptr) return;
 	progressBar->setValue(value);
 	progressBar->setFormat(s);
+	
```

Comment:
I didn't catch this earlier, but `setFormat()` accepts a format string, not a number. Not sure if your way triggers some undocumented compatibility mode perhaps?

Reply:
Hmm... that needs documentation from my side.

QProgressbar does not have a way of setting a text directly.
All I can do is setting a format, that is used to create the text.
By setting a format that does not contain a placeholder for the value, only the format string is shown.
By using a format string that is the way I want the string to be formatted, I get the result I want.

The alternative is, to overlay the progressbar with a label. Sounds messy.

Alternatively, I could look at QSlider or QScrollbar, but they do not have any text. (where I would like to have it)

Or customize the QProgressbar.